lib/plain/plain-client.ts

```diff
@@ -138,11 +138,11 @@ export const createPlainClient = (
       validate: wrap(wrapParams, 'BulkAction', 'validate'),
     },
     comment: {
-      get: wrap(wrapParams, 'Comment', 'get'),
-      getMany: wrap(wrapParams, 'Comment', 'getMany'),
-      create: wrap(wrapParams, 'Comment', 'create'),
-      update: wrap(wrapParams, 'Comment', 'update'),
-      delete: wrap(wrapParams, 'Comment', 'delete'),
+      get: plainWrap(wrapParams, 'comment', 'get'),
+      getMany: plainWrap(wrapParams, 'comment', 'getMany'),
+      create: plainWrap(wrapParams, 'comment', 'create'),
+      update: plainWrap(wrapParams, 'comment', 'update'),
+      delete: plainWrap(wrapParams, 'comment', 'delete'),
```

Comment:
I won't be able to use this as the `MRActions` is being used in a lot of other parts, specially in the `lib/adapters/REST/endpoints/comment.ts` so I need to update the normal non-plain client anyway, will move this PR to draft while I resolve this

Reply:
Touching this lib at the very core might be something where @andipaetzold might wanna help out or share his opinion :)